p2panda-js/openrpc.json

```diff
@@ -0,0 +1,150 @@
+{
+  "openrpc": "1.0.0-rc1",
+  "info": {
+    "title": "p2panda RPC interface",
+    "version": "0.0.1"
+  },
+  "components": {
+    "schemas": {
+      "EntryArgs": {
+        "type": "object",
+        "properties": {
+          "entryHashSkiplink": {
+            "type": "string"
+          },
+          "entryHashBacklink": {
+            "type": "string"
+          },
+          "seqNum": {
+            "type": "number"
+          },
+          "logId": {
+            "type": "number"
+          }
+        }
+      },
+      "Entry": {
+        "type": "object",
+        "properties": {
+          "author": {
+            "type": "string"
+          },
+          "entryBytes": {
+            "type": "string"
+          },
+          "entryHash": {
+            "type": "string"
+          },
+          "logId": {
+            "type": "number"
+          },
+          "payloadBytes": {
+            "type": "string"
+          },
+          "payloadHash": {
+            "type": "string"
+          },
+          "seqNum": {
+            "type": "number"
+          }
+        }
+      },
+      "Entries": {
+        "type": "array",
+        "items": {
+          "$ref": "#/components/schemas/Entry"
+        }
+      },
+      "QueryEntriesResponse": {
+        "type": "object",
+        "properties": {
+          "entries": {
+            "type": "array",
+            "items": {
+              "$ref": "#/components/schemas/Entries"
```

Comment:
Should this actually be:

`"$ref": "#/components/schemas/Entry"`

As it is we have an array of array of entries.

Reply:
You are right I think!

Reply:
You are right I think!